libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/route/line/api/MapboxRouteLineView.kt

```diff
@@ -212,16 +180,48 @@ class MapboxRouteLineView(options: MapboxRouteLineOptions) {
         rebuildSourcesAndLayersIfNeeded(style)
     }
 
+    /**
+     * Applies drawing related side effects.
+     *
+     * @param style a valid [Style] instance
+     * @param routeDrawData a [Expected<RouteLineError, RouteSetValue>]
+     * @param map [MapboxMap] which the routeDrawData should be drawn on
+     * @param callback [RoutesRenderedCallback] to be invoked when the routes are rendered
+     */
+    @ExperimentalPreviewMapboxNavigationAPI
+    fun renderRouteDrawData(
+        style: Style,
```

Comment:
The Style comes from the Map so it's not necessary to have this as a parameter.

Reply:
Yes, but it's nullable. The original method receives non-null style. But I think we can just do nothing if it's null.